src/imitation/scripts/analyze.py

```diff
@@ -74,62 +108,174 @@ def gather_tb_directories(
     return {"n_tb_dirs": tb_dirs_count, "gather_dir": tmp_dir}
 
 
+def _get_exp_command(sd: sacred_util.SacredDicts) -> str:
+    return str(sd.run.get("command"))
+
+
+def _get_algo_name(sd: sacred_util.SacredDicts) -> str:
+    exp_command = _get_exp_command(sd)
+
+    if exp_command == "train_adversarial":
+        algo = get(sd.config, "algorithm")
+        if algo is not None:
+            algo = algo.upper()
+        else:
+            print(sd.config)
```

Comment:
What's the purpose of this? Perhaps add a print statement before this to give some context?

Reply:
THis should probably be removed or otherwise editted. I think I added it here for debug purposes